test/k8sT/Updates.go

```diff
@@ -47,18 +47,23 @@ var _ = Describe("K8sUpdates", func() {
 		kubectl.Exec(fmt.Sprintf(
 			"%s delete --all pods,svc,cnp -n %s", helpers.KubectlCmd, helpers.DefaultNamespace))
 
+		kubectl.DeleteETCDOperator()
+
 		ExpectAllPodsTerminated(kubectl)
 	})
 
 	AfterAll(func() {
 		_ = kubectl.Apply(helpers.DNSDeployment())
 
+		// Deploy the etcd operator as it was removed by the delete --all
```

Comment:
Why this is needed on the AfterAll? Asserts on AfterAll are not good for how ginkgo works.

You can remove this one, because the next test will install this, and if fails logs will be collected, and happens on BeforeAll that it's a better place to happens. IMHO.